.github/workflows/package.yml

```diff
@@ -49,6 +50,24 @@ jobs:
     strategy:
       fail-fast: false
     steps:
+      - name: Check input
+        # This check the apis input. It detect whether the package is only asking for subset of
+        # of the products, like auth,firestore only etc. If the input apis are less than the
+        # default_apis env attribute, we determine this package only package subset and for 
+        # check up purpose only.
```

Comment:
Instead of "we determine..."  maybe "we package only the provided apis, for test purposes only."

Reply:
done